src/data/markdown/docs/30 xk6-browser/01 Get started/01 Welcome.md

```diff
@@ -38,33 +38,31 @@ The main use case for xk6-browser is to test performance on the browser level. B
 <!-- eslint-skip -->
 
 ```javascript
-import { check } from 'k6';
 import { chromium } from 'k6/x/browser';
```

Comment:
The import path will change to `k6/experimental/browser` after the merge. We probably should wait to publish this until after the merge and after `k6` have done their release (i think that's in mid-feb).